test/zipkin-simple.test.js

```diff
@@ -134,7 +134,7 @@ describe("Zipkin client", function () {
 				traceId: "test traceId",
 				spanId: "test spanId",
 				parentSpanId: "test parent_id",
-				timestamp: "test timestamp",
+				timestamp: new Date().getTime(),
```

Comment:
is this in microseconds? if it is milliseconds, multiply by 1000 (just to set a good example)